upgrade/1.2.1/upgrade_network.md

```diff
@@ -0,0 +1,102 @@
+# Upgrade and validate Switch Configurations
+
+## Prerequisites
+
+* SSH access to the switches or the running configuration file
+* Upgrade CANU: [Install/Upgrade CANU](../../operations/network/management_network/canu_install_update.md)
+  * Run `canu --version` to see version
+  * CANU installed with version 1.6.13 or greater.
+* Use the paddle file (CCJ) or validate SHCD [Validate SHCD](../../operations/network/management_network/validate_shcd.md)
+* System Layout Service (SLS) input file. [Collect Data](../../operations/network/management_network/collect_data.md)
+
+> **CAUTION:** All of these steps should be done using an out-of-band connection. This process is disruptive and will require downtime.
+
+## Generate Configuration Files
+
+Ensure that the correct architecture (`-a` parameter) is selected for the setup in use.
+
+The following are the different architectures that can be specified:
+
+* `Tds` – Aruba-based Test and Development System. These are small systems characterized by Kubernetes NCNs cabled directly to the spine.
+* `Full` – Aruba-based Leaf-Spine systems. These are usually customer production systems.
+* `V1` – Dell and Mellanox based systems of either a TDS or Full layout.
+
+Generating a configuration file can be done for a single switch, or for the full system. Below are example commands for both scenarios:
+
+**Important:** Modify the following items in your command:
+
+* `--csm` : Which CSM version configuration do you want to use? For example, `1.2` or `1.0`
```

Comment:
Add a follow-on sentence: "Only major and minor versions of CSM are tracked at this time.  CANU bugfixes are captured in the latest CANU version and do not align with CSM bugfix versions."